osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -2009,7 +2148,7 @@ def requires_time_period_schedule(task, task_runner, params):
 
 
 class ScheduleHandle:
-    def __init__(self, task_name, sched, task_progress_control, runner, params):
+    def __init__(self, task_allocation, sched, task_progress_control, runner, params):
         """
         Creates a generator that will yield individual task invocations for the provided schedule.
 
```

Comment:
The docstring should also be updated to reflect the change from task_name to task_allocation
```
:param task_name: The name of the task for which the schedule is generated.
```